Dzmitry_Skarabahatau/03.GIT.Hosting/README.md

```diff
@@ -0,0 +1,21 @@
+## 03.GIT.Hostingt
```

Comment:
There is no task "Automate pushing your changes to all remote repositories" from https://github.com/pluhin/sa.it-academy.by/wiki/03.-GIT.-Hosting